### DISCUSSION TITLE
Credits not recieved

### DISCUSSION THREAD

**Discussion by ashutosh887** [github/india#364] [General]
I have the GitHub and Azure account with the same Email ID
I tried to get access to azure and start my free trial but it isn't working

Can someone help me !

**Comment by TheSuranaverse** [comment]
A group for all to get notified for any **Official FRT Announcement, Discuss & Clear any Doubts, Learn & Complete** together.
One-stop place for daily **Events/Workshops + Hackathons + Internships/Jobs/Free Scholarship & Courses + 24/7 Live Music** & much more...

Link to join: https://discord.gg/drQ7ySCdX2
GitHub Solution Forum: https://github.com/github/india/discussions/353

Thank you 🙏
